### PR TITLE
Pre migration cloud test

### PR DESCRIPTION
**Pull request by Settis**

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/103/

**Comment by mirrorcoder** [comment]
PEP8 check?

**Comment by Settis** [comment]
Done.

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/108/

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/109/

**Comment by miarmak** [comment]
Maybe we should amend such commits?)

**Comment by Settis** [comment]
All commits merged as one.

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/110/

**Comment by miarmak** [comment]
Ok from my side.

**Comment by oleksii-shyman** [comment]
looks ok

File: cloud/cloud.py [hunk 1]
```diff
@@ -60,6 +61,9 @@ def make_cloud_config(config, position):
         for k, v in getattr(config, position + '_mysql').iteritems():
             cloud_config['mysql'][k] = v
 
+        for k, v in getattr(config, position + '_rabbit').iteritems():
+            cloud_config['rabbit'][k] = v
+
```
Comment (roman-verchikov) [inline cloud/cloud.py]: OMG :) Is there any way we stop adding these? :) All of them are *completely* identical other than the attribute name. Probably OK for now, but needs refactoring in future.

**Comment by roman-verchikov** [comment]
+1

**Comment by Wasapl** [comment]
Build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/120/

**Comment by Settis** [comment]
I can't update it. Please see https://github.com/MirantisWorkloadMobility/CloudFerry/pull/123

**Pull request closed without merging**